mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/DelayedLambdaType.cs

```diff
@@ -0,0 +1,170 @@
+﻿﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Mono.Debugger.Soft
+{
```

Comment:
Why are these types here ? They are not used by the rest of the assembly.

Reply:
Removed